src/RokuAdapter.ts

```diff
@@ -1010,6 +1022,21 @@ export class RokuAdapter {
             } while (commandsExecuted < 10);
         }
     }
+
+    /**
+     * Passes the debug functions used to locate the client files and lines to the RendezvousTracker
+     */
+    public setRendezvousDebuggerFileConversionFunctions(convertDebuggerLineToClientLine, convertDebuggerPathToClient) {
```

Comment:
Can you please define the types for these parameters. ` (debuggerPath: string, lineNumber: number)=> number` and `(debuggerPath: string) => string` (I think).

Reply:
Done and addressed in the RendezvousTracker as well.